src/Collector.cpp

```diff
@@ -363,3 +366,36 @@ void CollectorTime::setValue(int i, const Token& t) {
   }
 
 }
+
+void CollectorDeparsed::setValue(int i, const Token& t) {
+  switch(t.type()) {
+  case TOKEN_STRING: {
+    boost::container::string buffer;
+    SourceIterators string = t.getString(&buffer);
+    std::string std_string(string.first, string.second);
+    RObject ans = R_NilValue;
+
+    ParseStatus status;
+
+    CharacterVector c(std_string);
+    ExpressionVector r = R_ParseVector(c, -1, &status, R_NilValue);
+    if (status == PARSE_OK) {
+      for(R_len_t i = 0; i < Rf_length(r); i++) {
+        ans = Rf_eval(VECTOR_ELT(r, i), R_GlobalEnv);
+      }
+
+      SET_VECTOR_ELT(column_, i, ans);
+    } else {
+      // parse error
+      Rcpp::stop("Invalid expression %s", c);
```

Comment:
This should probably be a "problem" rather than terminating parsing of the whole file.